libr/main/rasm2.c

```diff
@@ -47,6 +47,7 @@ static void __as_free(RAsmState *as) {
 	r_asm_free (as->a);
 	r_anal_free (as->anal);
 	r_lib_free (as->l);
+    	R_FREE (as);
```

Comment:
just call free, no need to nullify because its not used outside the function

Reply:
True it was just to counter R_NEW0 style-wise :-)